autopush/src/server/mod.rs

```diff
@@ -159,14 +159,19 @@ impl AppState {
     pub fn from_settings(settings: Settings) -> Result<Self> {
         let crypto_key = &settings.crypto_key;
         if !(crypto_key.starts_with('[') && crypto_key.ends_with(']')) {
-            return Err(ApcErrorKind::GeneralError("Invalid AUTOPUSH_CRYPTO_KEY".into()).into());
+            return Err(
```

Comment:
Took the opportunity to fix up these so they're more helpful:
* Include the proper env key name
* show invalid key (say, if it's mangled because the test script is dumping `b'somestring'`)